js/classes/game_engine.js

```diff
@@ -107,7 +107,7 @@ class GameEngine {
         // Clear the whole canvas with transparent color (rgba(0, 0, 0, 0))
         CTX.clearRect(0, 0, CANVAS.width, CANVAS.height);
         CTX.fillStyle = "#00ff00";
-        CTX.fillRect(DIMENSION.MIN_X - CAMERA.pos.x, DIMENSION.MIN_Y - CAMERA.pos.y, DIMENSION.BLOCK_WIDTH * Block.SCALED_SIZE, DIMENSION.BLOCK_HEIGHT * Block.SCALED_SIZE);
+        CTX.fillRect(0, 0, Camera.SIZE.x, Camera.SIZE.y);
```

Comment:
Very nice. I like the simplicity. Much more readable. @B1GD3VD0G

Reply:
No reason to fill in the whole dimension when you could just fill in the whole camera, lol

Reply:
And honestly, that code needs to go anyway lol. But it looks so bad on the gray. We'll have background colors drawn in when we have actual zones.